src/chargen.cpp

```diff
@@ -1084,6 +1118,37 @@ static void start_game(descriptor_data *d, const char *origin)
     GET_LOADROOM(d->character) = RM_CHARGEN_START_ROOM;
   }
 
+  // Otaku get some starting gear, so assign that here
+  if (d->ccr.is_otaku) {
+    // Equip cyberware (deduct essence and modify stats as appropriate)
+    obj_data *temp_obj;
+    for (int cyb = 0; cyb < OTAKU_CYBERWARE_NUM; cyb++) {
+      if (otaku_cyberware[cyb]) {
+        if (!(temp_obj = read_object(otaku_cyberware[cyb], VIRTUAL, OBJ_LOAD_REASON_ARCHETYPE))) {
+          snprintf(buf, sizeof(buf), "SYSERR: Invalid cyberware item %ld specified for otaku class.",
+                  otaku_cyberware[cyb]);
+          mudlog(buf, CH, LOG_SYSLOG, TRUE);

```

Comment:
FYI, this is the old style of logging. You don't need to change this, but we now would skip pre-composing the buf and just use:

`mudlog_vfprintf(CH, LOG_SYSLOG, "fmt", vars);`